R/uvhydrograph-data.R

```diff
@@ -241,7 +241,8 @@ getUvHydro <- function(ts, field, estimatedOnly = FALSE){
   
   if(!is.null(y) & !is.null(x)){
 
-    time <- as.POSIXct(strptime(x, "%FT%T"))
+    format <- "Ymd HMOS z"
+    time <- parse_date_time(x,format,tz = Sys.timezone(),quiet=TRUE)
```

Comment:
I'm fairly sure the location of the R server is how the data gets its UTC Offset, but want to test on dev.